org.mwc.cmap.TimeController/src/org/mwc/cmap/TimeController/views/TimeController.java

```diff
@@ -610,7 +611,13 @@ else if (DEBRIEF_S_WINDOW.equals(preferenceStore.getString(
       {
         areaToRecord = getScreenArea(getViewSite().getShell());
       }
-      return areaToRecord;
+      else if (DEBRIEF_PLOT_WINDOW.equals(preferenceStore.getString(
+          P_SCREEN_AREA)))
+      {
+        

```

Comment:
@saulhidalgoaular - in this `if` structure, should it be retrieving the viewport of the current editor?  Oh, and it shouldn't include the tabs at the top. So, it's the content pane of the current editor, not the whole of the current editor - please.

Reply:
Correct. It is still not implemented. I need to find out how to calculate the viewport area. I pushed it partially to fix the bug mentioned in the comment above. 😢

Reply:
>  I need to find out how to calculate the viewport area.

I think it's something like this:

https://stackoverflow.com/a/11780445/92441